File: 1441-build-an-array-with-stack-operations/1441-build-an-array-with-stack-operations.cpp
class Solution {
public:
    vector<string> buildArray(vector<int>& target, int n) {
        vector<string>ans;
        int counter=1;
        for(int i=0;i<target.size();i++){
            ans.push_back("Push");
            if(target[i]!=counter){
                ans.push_back("Pop");
                
                i--;
            }
            counter++;
        }
        return ans;
        
    }
};

File: 1441-build-an-array-with-stack-operations/1441-build-an-array-with-stack-operations.py
class Solution(object):
    def buildArray(self, target, n):
        """
        :type target: List[int]
        :type n: int
        :rtype: List[str]
        """
        ans=[]
        for i in range(1,target[-1]+1):
            ans.append("Push")
            if i not in target:
                ans.append("Pop")
        return ans